packages/database/src/core/BaseDatabase.js

```diff
@@ -114,9 +114,9 @@ export class BaseDatabase {
     await this.connectionPromise;
   }
 
-  async fetchSchemaForTable(databaseRecord) {
+  async fetchSchemaForTable(databaseRecord, schemaName) {
```

Comment:
This change is because I have the debug_logs table in `logs` schema, and we've only been using the `public` schema in tupaia